src/odemis/driver/piezomotor.py

```diff
@@ -365,9 +417,9 @@ def _waitEndMotion(self, f, targets):
             if target is not None:
                 move_length = max(abs(self.position.value[ax] - target), move_length)
         if move_length == 0:  # open loop
-            move_length = STROKE_RANGE
+            move_length = STROKE_RANGE[1] - STROKE_RANGE[0]
         dur = move_length / self._speed
-        max_dur = max(dur * 2, 0.1)  # wait at least 0.1 s
+        max_dur = max(dur * 2, 0.5)  # wait at least 0.5 s
```

Comment:
Usually, I just do `dur * 2 + 1`, so it's really pessimistic, and don't risk timing out by mistake.